client/src/Tools/_framework/Paths/Activities.tsx

```diff
@@ -132,6 +132,12 @@ export async function action({ request, params }) {
       name,
     });
     return true;
+  } else if (formObj._action == "add classification") {
```

Comment:
Add this to `ActivityViewer.tsx` as well so it works from there, too. (I'm working on a PR where this will no longer be necessary.)

Reply:
Done, added to <code>ActivityEditor.tsx</code>